gammapy/cube/make.py

```diff
@@ -257,3 +297,359 @@ def _check_selection(selection):
+
+        Parameters
+        ---------
+
+        observations: `~gammapy.data.Observations`
+            Observations to process
+        spectrum : `~gammapy.spectrum.models.SpectralModel`, optional
+            Spectral model to compute the weights.
+            Default is power-law with spectral index of 2.
+        keepdims : bool, optional
+            If this is set to True, the energy axes is kept with a single bin.
+            If False, the energy axes is removed
+
+        Returns
+        ---------
+        maps: list of dict of maps. Each observation will have the following maps
+            counts: The counts map
+            background_irf: The template background map from the IRF
+            exposure: The uncorrelated on exposure map
+            exposure_off: The off exposure map convolved with the ring
+            alpha: The alpha map (1/exposure map)
+            off: The off map
+            background_ring: The ring background map ( = alpha * off)
+
+        """
+        return self._run(
+            observations, sum_over_axis=True, spectrum=spectrum, keepdims=keepdims
+        )
+
+    def run(self, observations):
+        """Returns a list of dictionaries of 3D maps
+        Significance and excess can be computed for each slice
+
+        Returns
+        ---------
+        maps: list of dict of maps. Each observation will have the following maps
+            counts: The counts map
+            background_irf: The template background map from the IRF
+            exposure: The uncorrelated on exposure map
+            exposure_off: The off exposure map convolved with the ring
+            alpha: The alpha map (1/exposure map)
+            off: The off map
+            background_ring: The ring background map ( = alpha * off)
+
+        """
+        return self._run(observations, sum_over_axis=False)
+
+    def significance_map(self, maplist, convolution_radius=None):
+        """
+        Parameters
+        ---------
+        maplist: the list of dictionaries returned by MapMakerRing.run()
+                or MapMakerRing.run_images()
+        convolution_radius: `~astropy.units.Quantity`
+            The convolution radius for Tophat2DKernel
+
+            If not specified, returns the uncorrelated significance map
+
+        Returns
+        -------
+        significance_map: `~gammapy.maps.Map`
+        Stacked significance map for the entire region.
+
+        """
+        if convolution_radius:
+            scale = self.geom.pixel_scales[0].to("deg")
+            theta = (convolution_radius * scale).value
+        else:
+            theta = 1
+        tophat = Tophat2DKernel(theta)
+        tophat.normalize("peak")
+        selection = ["on", "off", "exposure_on", "exposure_off"]
+
+        stacked_map = {}
+        for name in selection:
+            stacked_map[name] = Map.from_geom(maplist[0]["counts"].geom)
+
+        for amap in maplist:
```

Comment:
OK, I see, the stacking is implemented here. But how can the user just get a stacked counts or background map?